### ISSUE TITLE
flang1, flang2: mp.h: MP_TASK_MERGEABLE defined twice

### ISSUE THREAD

**Issue by pawosm-arm** [flang-compiler/flang#213]
Found this looking at 'task' clauses definitions in mp.h (both flang1 and flang2 directories):

```
#define MP_TASK_IMMEDIATE 0x40
#define MP_TASK_MERGEABLE 0x80
#define MP_TASK_MERGEABLE 0x80
#define MP_TASK_PRIORITY 0x100
```

**Comment by pawosm-arm** [comment]
Doesn't seem to be used anywhere.

**Comment by sscalpone** [comment]
Fixed in 5c504a6be0d603c253d2f7ea23bdc0a804562466.  It'll be used eventually.  Also, at some point, mp.h will be shared instead of duplicated.
